surveys/2025/contributor-survey/questions.md

```diff
@@ -0,0 +1,230 @@
+- Contracting for a company (I'm paid *primarily* to work on the Rust Project)
+- Contracting for a company (work on the Rust Project is an irregular or a side part of my job)
+- Individual sponsors (e.g. GitHub sponsors, Patreon, ...)
+- Grants (e.g. Rust Project Grants, Sovereign Tech Fund, ...)
+- Other (open response)
+
+### How long have you been funded for your Rust Project contributions?
+
+If you receive funds for your Rust Project contributions from a stable source, such as a contract or an employment, please tell us how long have you been *continuously* funded for (not necessarily from the same employer/contract).
+
+Type: select one (optional)
+
+- My funding started this year
+- Since 2024
+- Since 2023
+- Since 2022
+- Since 2021
+- Since 2020
+- Since 2019 or before
+
+### When does your contract/employment end?
+
+If you have a contract or if you are employed to work on Rust, please tell us when is your contract/employment ending (even if it might get refreshed for another term).
+
+Type: select one (optional)
+
+- This year (2025)
+- 2026
+- 2027
+- 2028
+- 2029 or later
+- My contract/employment has no end date
+- I don't know
+- Not applicable to my source of funding
+- Other (open response)
+
+### Are your contributions directed by your source of funding?
+
+We want to find out whether the entity/entities that pay for your contributions determine what contributions you make.
+
+Type: select one (optional)
+
+- Yes, my source of funding tells me what contributions I should make
+- No, I decide what I will contribute to
+- Other (open response)
+
+### Would you appreciate being paid for contributing to the Rust Project?
+
+Type: select one (optional)
+
+- Receiving more funds would allow me to keep my current level of contributions
+- Receiving more funds would allow me to increase my current level of contributions
+- I am happy with the current funding that I have at the moment
+- I do not require/want any funding for my contributions
+- Other (open response)
+
+### How much money would you like to be paid for contributing to the Rust Project?
+
+What is a minimum **hourly** rate that could comfortably sustain your contributions to the Rust Project, assuming that it would be your only source of funding, and you would be working full-time on Rust?
+
+Note: we ask for a minimum to establish some baseline. Of course, everyone would like to get paid as much as possible :)
+
+Please provide the value as an integer in $USD (US Dollars).
+
+Type: free form (optional)
+
+### Would you potentially like to be included on a public list of contributors looking for funding?
+
+If we ever had something like a public list of Rust Project contributors looking for funding, what would you think of that?
+
+Type: select one (optional)
+
+- That sounds like a good idea, and I would potentially like to be on that list
+- That sounds like a good idea, but I do not want to be on such a list
+- I'm neutral/I don't know
+- That sounds like a bad idea
```

Comment:
maybe?

```suggestion
- That sounds like a bad idea (please explain why)
```

Reply:
Good point, I was also thinking about that, but I'm not sure how to combine "select one answer" + allow an open answer, hmm.

Reply:
I tried this as a compromise: https://github.com/rust-lang/surveys/pull/350/commits/fac52ae1bea59d1d3342f4c63218529bdc6256d9

Reply:
The wording is fine but if I read correctly there is no text field where people can write ~~things~~ the "why", right?